_includes/release.html

```diff
@@ -1,157 +1,188 @@
 <h3 class="section">Series</h3>
 <div class="row">
-  <div class="col-sm-4">
-    <div class="series raised">
-      <div class="series-banner">
-        <h4 class="series-label">
-          <div class="span label label-stable">stable</div>
-        </h4>
-      </div>
-      <div class="clear"></div>
-      <h3 class="version">1.2</h3>
-      <h6 class="version-date">2020-09-24</h6>
-      <div class="description">Align code with Quarkus 1.9</div>
-      <div class="more-info">
-        <a class="btn btn-primary" href="/releases/1.2/">
-          More info&nbsp;&nbsp;&nbsp;&nbsp;
-          <span class="icon-arrow-right"></span>
-        </a>
+  {% for version in site.data.versions.versions %}
+   {% assign formatedVersion = version | replace: ".", "" %} 
+   {% assign releaseDetails = site.data.releases[formatedVersion] %} 
+   {% if releaseDetails.series.displayed %}
+    <div class="col-sm-4">
+      <div class="series raised">
+        <div class="series-banner">
+          <h4 class="series-label">
+            <div class="span label label-stable">stable</div>
+          </h4>
+        </div>
+        <div class="clear"></div>
+        <h3 class="version">{{ version }}</h3>
+        <h6 class="version-date">2020-09-24</h6>
+        <div class="description">Align code with Quarkus 1.9</div>
```

Comment:
Those values should be obtainted from the data files, not be hard-coded.

Reply:
@gunnarmorling - @indraraj was aware that there was still work to do - I PR'd some additional changes for him to review

Reply:
Excellent, thx!